root/etc/e-smith/db/configuration/migrate/delete_agent

```diff
@@ -0,0 +1,6 @@
+{
+    # Disable the agent service
+    $DB->get_value_and_delete('agent') || return '';
+    system('systemctl', 'disable', '--now', 'agent');
```

Comment:
This is not needed: the system should stop the agent by itself.

Reply:
This is quite strange, esmith should check the default db value.
I'd still remove the disable call: only very few machines have this package installed.